packages/website/components/account/paymentTable.js/paymentTable.js

```diff
@@ -0,0 +1,81 @@
+import Button from '../../button/button.js';
+
+const PaymentTable = ({ plans, currentPlan, setPlanSelection, setIsPaymentPlanModalOpen }) => {
+  return (
+    <>
+      {currentPlan && (
+        <p className="billing-content-intro">
+          <span>
+            Your current plan is: <strong>{currentPlan.title}</strong>
+          </span>
+          <small>Billing Cycle: Aug 18 - Sept 18</small>
```

Comment:
This probably shouldn't be hard-coded :stuck_out_tongue:

Reply:
Oy. Good catch. Thank you!